isp/network/interfaces.d/eth1

```diff
@@ -0,0 +1,6 @@
+allow-hotplug eth1
+
+iface eth1 inet manual 
+iface eth1 inet6 auto
```

Comment:
Use ipv6 instead of ipv4. Run ipv6 in auto configuration mode to configure address on ipv6 link.